libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortBar.tsx

```diff
@@ -127,11 +128,11 @@ export class CohortBar extends React.Component<
     });
   };
 
-  private onCohortChange = (newCohort: Cohort): void => {
+  private onCohortChange = (newCohort: Cohort | DatasetCohort): void => {
```

Comment:
will remove Cohort eventually?

Reply:
yes, as mentioned in the meeting, will remove it once explanation dashboard is deprecated